packages/components/breadcrumb/src/breadcrumb.ts

```diff
@@ -1,35 +1,19 @@
-import { buildProps } from '@puik/utils'
-import type { RouteLocationRaw } from 'vue-router'
-import type { ExtractPropTypes, PropType } from 'vue'
-import type Breadcrumb from './breadcrumb.vue'
+import type { RouteLocationRaw } from 'vue-router';
+import type Breadcrumb from './breadcrumb.vue';
+import type { PuikLinkTargetVariants } from '../../link';
 
-import type { PuikTargetString } from '../../link/src/link'
 export interface BreadcrumbItem {
   label: string
   to?: RouteLocationRaw
   href?: string
-  target?: PuikTargetString
+  target?: `${PuikLinkTargetVariants}`
   dataTest?: string
 }
 
-export const breadcrumbProps = buildProps({
-  items: {
-    type: Array as PropType<BreadcrumbItem[]>,
-    required: false,
-    default: () => [],
-  },
-  separatorIcon: {
-    type: String,
-    required: false,
-    default: 'keyboard_arrow_right',
-  },
-  icon: {
-    type: String,
-    required: false,
-    default: undefined,
-  },
-} as const)
-
-export type BreadcrumbProps = ExtractPropTypes<typeof breadcrumbProps>
+export interface BreadcrumbProps {
+  items: BreadcrumbItem[]
```

Comment:
items were optional before. It was a mistake ? (Indeed, use a breacrumb without items it's useless)

Reply:
I think it should be mandatory, you can't use a breadcrumb without items, it was causing some TS errors when migrating to defineProps

Reply:
Yes, I agree with you.